modules/spring-config/src/main/java/org/mule/runtime/config/internal/model/ApplicationModel.java

```diff
@@ -1041,19 +1036,6 @@ private ComponentModel getOnErrorComponentModel(ComponentModel onErrorModel) {
     }
   }
 
-  private void validateExceptionStrategyWhenAttributeIsOnlyPresentInsideChoice() {
```

Comment:
Who is doing this validation now?

Reply:
this was leftovers from mule 3, mule 4 uses error-handler, not exception strategy